doc/_templates/respirator/src/dlite.rst.j2

```diff
@@ -0,0 +1,4 @@
+{% extends "doxygenfile.rst.j2" %}
+{% block description %}
+DDDDDDLITE
```

Comment:
Should there be so many D's?

Reply:
It was a test to check if custom comments worked. it did. I will remove this :)